sogs/utils.py

```diff
@@ -15,6 +15,12 @@ def message_body(data: bytes):
     msg.ParseFromString(remove_session_message_padding(data))
     return msg.dataMessage.body
 
+def message_contents(data: bytes):
```

Comment:
would be better to call this `parse_message` or something of that kind, and have the function above call that function.